examples/spark-app-demo/k8s/__init__.py

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
@assapin you can start with the k8s example 
(i will add readme soon, meanwhile, you can use the getting_started doc with examples/spark-app-demo/k8s files)